src/Integration.Vsix/CFamily/PortedFromJava/MsvcDriver.cs

```diff
@@ -372,6 +372,10 @@ public static Request ToRequest(CFamilyHelper.Capture[] captures)
                     {
                         // if major > 19 we rely on /std.  By default, /std:c++14 is specified
                         predefines.Append("#define _HAS_CHAR16_T_LANGUAGE_SUPPORT 1\n");
+                        if (minor > 23)
```

Comment:
Is the define expected to be included if the compiler version major version is 20?

Reply:
Discussed offline - behaviour is unknown -> assume the macro won't be defined.